src/lib/Poster.svelte

```diff
@@ -121,6 +125,7 @@
           {title}
         {/if}
       </h2>
+      <time>{year}</time>
```

Comment:
This might look better if its put next to the title of the movie (like how it is in the movie or tv page title):

![image](https://github.com/sbondCo/Watcharr/assets/37304121/130acaea-8880-4da7-9809-d7b2a1f62f1a)

Reply:
Great, done!